src/event/log_event.rs

```diff
@@ -337,50 +341,6 @@ impl Serialize for LogEvent {
     }
 }
 
-impl vrl::Target for LogEvent {
```

Comment:
Dropped this in lieu of the implementation on `VrlTarget` adapter in `vrl_target.rs`.